README.md

```diff
@@ -1,77 +1,103 @@
 # pika
+
 ORM-like SQL builder inspired by [kayak/pypika](https://github.com/kayak/pypika)
 
+---
+
+# Notice of Deprecation
+
+This repository contains the original code for pika, which was developed by CIQ as
+a proof of concept (POC). It is intended solely as an illustrative example and is
+provided as-is for reference purposes only.
+
+## Disclaimer
```

Comment:
You could replace this section with the below statement, but I see value in a less legal jargon and more explanatory structure. I'll leave that decision to you:

> THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.